src/lib.rs

```diff
@@ -71,6 +72,15 @@ ast_struct! {
     pub struct ItemFn {
         pub attrs: Vec<Attribute>,
         pub vis: Visibility,
+        pub sig: Signature,
+        pub block: Block,
```

Comment:
Could you please use `Box<Block>` here? This is the way it's done in syn.

Reply:
done.